### DISCUSSION TITLE
Is anyone using the Alexa Emergency Assist subscription?

### DISCUSSION THREAD

**Discussion by danielbrunt57** [alandtse/alexa_media_player#2653] [Q&A]
Alexa Guard was discontinued about a year ago and replaced it with a subscription model - Alexa Emergency Assist.
Is there anyone using the new subscription who can confirm the "guard" device & alarm panel for Home/Away modes is still working?

**Comment by alandtse** [comment]
This really should be a discussion. Also, forcing past the template will just encourage others to try. I think it's smart enough to auto close them, but who knows.
